### PR TITLE
build: use issue forms for issue creation

### PR DESCRIPTION
**Pull request by lekhmanrus**
* [x] Use issue forms from Github for creating issues
* [x] Add pull request Github template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+name: 🐞 Bug report
```
Comment (josephperrott) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:1]: As part of this migration/clean up we need to also remove all of the usage of emoji characters in our templates.  They are not good for accessibility and should be removed from their usage here.  From looking through the changes, it looks like none of the emojis need replacing, and instead can just be removed.

This is true for all of the files in the change

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        - extract-i18n
+        - generate
+        - help
+        - lint
+        - new
+        - run
+        - serve
+        - test
+        - update
+        - version
```
Comment (josephperrott) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:35]: We should include an other option as well

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        * related code changes
+
+
+        If reproduction steps are not enough for reproduction of your issue, please create a minimal GitHub repository with the reproduction of the issue.
+        A good way to make a minimal reproduction is to create a new app via `ng new repro-app` and add the minimum possible code to show the problem.
+        Share the link to the repo below along with step-by-step instructions to reproduce the problem, as well as expected and actual behavior.
+
+        Issues that don't have enough info and can't be reproduced will be closed.
+
+        You can read more about issue submission guidelines [here](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue).
```
Comment (josephperrott) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:78]:
```suggestion
        You can read more about issue submission guidelines [here](https://github.com/angular/angular-cli/blob/main/CONTRIBUTING.md#-submitting-an-issue).
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 📚 Docs or angular.io issue report
+    url: https://github.com/angular/angular/issues/new
+    about: Report an issue in Angular's documentation or angular.io application
+  - name: ⚠️ Security issue disclosure
+    url: https://angular.io/guide/security#report-issues
+    about: Report a security issue in Angular Framework, Material, or CLI
+  - name: ❓ Support request
+    url: https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#question
```
Comment (josephperrott) [inline .github/ISSUE_TEMPLATE/config.yml:10]:
```suggestion
    url: https://github.com/angular/angular-cli/blob/main/CONTRIBUTING.md#question
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    about: Questions and requests for support.
+  - name: ⚡Angular Framework
+    url: https://github.com/angular/angular/issues/new
```
Comment (josephperrott) [inline .github/ISSUE_TEMPLATE/config.yml:13]:
```suggestion
    url: https://github.com/angular/angular/issues/new/choose
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    about: Issues and feature requests for Angular Framework
+  - name: 💎 Angular Material
+    url: https://github.com/angular/components/issues/new
```
Comment (josephperrott) [inline .github/ISSUE_TEMPLATE/config.yml:16]:
```suggestion
    url: https://github.com/angular/components/issues/new/choose
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
+## PR Checklist
+
+Please check if your PR fulfills the following requirements:
+
+<!-- Please check the one that applies using "x". -->
+
+- [ ] The commit message follows our guidelines: https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-commit-message-guidelines
```
Comment (josephperrott) [inline .github/PULL_REQUEST_TEMPLATE.md:7]:
```suggestion
- [ ] The commit message follows our guidelines: https://github.com/angular/angular-cli/blob/main/CONTRIBUTING.md#-commit-message-guidelines
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
+## PR Checklist
+
+Please check if your PR fulfills the following requirements:
+
+<!-- Please check the one that applies using "x". -->
```
Comment (josephperrott) [inline .github/PULL_REQUEST_TEMPLATE.md:5]:
```suggestion
<!-- Please check all that apply using "x". -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
+## PR Checklist
+
+Please check if your PR fulfills the following requirements:
```
Comment (josephperrott) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
Please check to confirm your PR fulfills the following requirements:
```

**Review by josephperrott — CHANGES_REQUESTED** [review]
Thanks for this cleanup, just a few tweaks but overall definitely what we are looking to do!

File: .github/ISSUE_TEMPLATE/1-bug-report.yml (hunk 1 above)
Comment (alan-agius4) [reply .github/ISSUE_TEMPLATE/1-bug-report.yml]: Looks like there are still some emojis around.

**Review by josephperrott — APPROVED** [review]
LGTM

**Pull request merged**